pkg/helm/client/client.go

```diff
@@ -134,3 +156,44 @@ func (c *ownerRefInjectingClient) Build(reader io.Reader, validate bool) (kube.R
 	}
 	return resourceList, nil
 }
+
+func SupportsOwnerReference(restMapper meta.RESTMapper, owner, dependent runtime.Object) (bool, error) {
```

Comment:
Put this function in `pkg/k8sutil/k8sutil.go` so we (and SDK users) can re-use it.